Cargo.toml

```diff
@@ -20,6 +20,9 @@ categories = [
 maintenance = { status = "actively-developed" }
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+wasm-bindgen = ["rand/wasm-bindgen"]
```

Comment:
@Zireael-N does this relate to your other pull?

Reply:
I'd like to better understand how to catch these errors. I'm new to the wasm world so I may do something run but wouldn't this catch it? https://github.com/softprops/again/blob/928e06c16c3748e3b24815663ed18e42133f5aee/.github/workflows/main.yml#L46

Reply:
It's a runtime error.

Reply:
I think, the easiest thing you can do is to make CI build a wasm binary and then attempt to interface with it via NodeJS to see if it throws an exception.

Reply:
Actually, this won't work with NodeJS because `wasm-timer` assumes that it's being run inside a browser and expects `window` to be available. :(

`wasm-bindgen`, according to [its guide](https://rustwasm.github.io/docs/wasm-bindgen/wasm-bindgen-test/index.html), can run tests using headless browsers but I've never done that.

Reply:
thanks for the tips

Reply:
this thread might be helpful. it seems possible to get rand working with wasm bindgen. I'll do some research https://github.com/rust-random/rand/issues/616

Reply:
It does work if you use a `wasm-bindgen` feature. The features I added in this PR just tell cargo to enable corresponding features while compiling `rand`.

Reply:
Aha https://github.com/rust-random/rand/issues/810#issuecomment-497930233 I was missing a bit of context here. Can you merge master to fix the merge conflict. I'll merge this aftet

Reply:
Rebased and force-pushed.

Reply:
I'll follow up on figuring out what an integration test looks like this in ci in https://github.com/softprops/again/issues/4